whois-commons/src/main/java/net/ripe/db/whois/common/rpki/WhoisRoaChecker.java

```diff
@@ -11,10 +11,14 @@
 import java.util.List;
 import java.util.Map;
 import java.util.Optional;
+import java.util.function.Predicate;
+import java.util.stream.Stream;
 
 import static net.ripe.db.whois.common.rpki.ValidationStatus.INVALID;
 import static net.ripe.db.whois.common.rpki.ValidationStatus.INVALID_ORIGIN;
+import static net.ripe.db.whois.common.rpki.ValidationStatus.INVALID_PREFIX_AND_ORIGIN;
```

Comment:
Document that `WhoisRoaChecker` validates *overlaps*. That makes it different from other `RpkiRoaChecker` implementations.

Also consider renaming `RpkiRoaChecker` as it's confusing with `WhoisRoaChecker` (i.e. which one is abstract?)

Reply:
well WhoisRoaChecker is extending RpkiRoaChecker so the abstract class is RpkiRoaChecker
I can add "Abstract" to make it more clear

Reply:
Thanks